<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageIntegrityController</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="main_layout">
    <item>
     <layout class="QVBoxLayout" name="common_layout">
      <item>
       <layout class="QHBoxLayout" name="input_file_layout">
        <item>
         <widget class="QLabel" name="input_file_label">
          <property name="text">
           <string>Input file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_file_path_lineedit"/>
        </item>
        <item>
         <widget class="QPushButton" name="choose_input_file_pushbutton">
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="output_folder_layout">
        <item>
         <widget class="QLabel" name="output_folder_label">
          <property name="text">
           <string>Output folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="output_folder_path_lineedit"/>
        </item>
        <item>
         <widget class="QPushButton" name="choose_output_folder_pushbutton">
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="block_size_layout">
        <item>
         <widget class="QLabel" name="block_size_label">
          <property name="text">
           <string>Block side (pixels):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="block_size_combobox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="spinbox_layout">
      <item>
       <widget class="QLabel" name="spinbox_label">
        <property name="text">
         <string>Noise margin factor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinbox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="action_layout">
      <item>
       <widget class="QPushButton" name="perform_pushbutton">
        <property name="text">
         <string>Perform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="signing_radiobutton">
        <property name="text">
         <string>Signing</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="verifying_radiobutton">
        <property name="text">
         <string>Verifying</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsview"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
